README.md

```diff
@@ -215,6 +215,7 @@ Describes the current state of the network. It is an object with these propertie
 | Property              | Type                                    | Description                                                                                        |
 | --------------------- | --------------------------------------- | -------------------------------------------------------------------------------------------------- |
 | `type`                | [`NetInfoStateType`](#netinfostatetype) | The type of the current connection.                                                                |
+| `isWifiEnabled`                | [`boolean`](#netinfostatetype) | Whether the device's WiFi is ON or OFF.                                                                |
```

Comment:
```suggestion
| `isWifiEnabled`                | [`boolean`](#netinfostatetype) | Whether the device's WiFi is ON or OFF.                                                                |
| `isWifiEnabled`                | `boolean`                      | Whether the device's WiFi is ON or OFF.                                                                |
```

Reply:
Can you also move this down to below the `isInternetReachable` line as those two are more important.